DTS-STN/passport-status
# Update splash page

## [ADO-1147](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1147)

### Description
This PR updates the splash page so it has a blurred background image (the same one that is on Canada.ca), centers the language selection box on the screen and updates the light gray color to be in line with Canada.ca.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure splash page looks and works well
